src/yoroi-wallets/types/other.ts

```diff
@@ -484,3 +493,19 @@ export type CommonMetadata = {
   readonly longName: null | string
   readonly maxSupply: null | string
 }
+
+export type MultiAssetRequest = {
```

Comment:
dont use Request or Response in type names

Reply:
why not `Array<{
    nameHex: string
    policy: string
  }` ?

Reply:
Almost every function in api.ts seems to use a type ending with `Request`, as this type is solely used to communicate with the backend and has no other purpose. I guess we can get rid of this when the backend is typed (which I think was discussed earlier), or do you have any other suggestions?